spec/widgets/category/category-widget-model.spec.js

```diff
@@ -32,6 +33,23 @@ describe('widgets/category/category-widget-model', function () {
     });
   });
 
+  describe('search state', function () {
```

Comment:
Yep, the change is only related with the `selected` attribute. Should we move it then?